dzgui.sh

```diff
@@ -417,11 +422,9 @@ check_map_count(){
             logger WARN "User aborted password prompt"
             return 1
         fi
-        local ct=$(sudo -S <<< "$pass" sh -c "sysctl -q vm.max_map_count | awk -F'= ' '{print \$2}'")
         logger DEBUG "Old map count is $ct"
```

Comment:
This logger line should be irrelevant if the $ct variable is unused

Reply:
Missed that one. I'll rip it out asap.

Reply:
Ok, afaik I removed it already?

Reply:
It might be a visual bug in the diff on GitHub's side. It looks like line 425 calls the logger to print the contents of `$ct`, but this variable is no longer being used? Sorry if I misread something.

Reply:
Oh I see it now. The line numbers are confusing:
![image](https://github.com/user-attachments/assets/a6b8c1ac-69c8-4f56-8205-3e12f13cd258)